livekit-plugins/livekit-plugins-hume/livekit/plugins/hume/tts.py

```diff
@@ -0,0 +1,422 @@
+            instant_mode (NotGivenOr[bool]): Enables ultra-low latency streaming.
+                Reduces time to first audio chunk, recommended for real-time applications.
+                Note: Incurs 10% higher cost when enabled.
+            strip_headers (NotGivenOr[bool]): If enabled, the audio for the chunks of a generation.
+                Once concatenated together, will constitute a single audio file.
+                If disabled, each chunk’s audio will be its own audio file, each with its headers.
+        """
+
+        if is_given(utterance_options):
+            # text is required in PostedUtterance but it is declared as an empty string
+            # it will be overwritten when input tokens are received
+            self._opts.utterance_options = PostedUtterance(
+                description=utterance_options.description if utterance_options.description else "",
+                voice=utterance_options.voice if utterance_options.voice else DEFAULT_VOICE,
+                speed=utterance_options.speed if utterance_options.speed else 1,
+                trailing_silence=utterance_options.trailing_silence
+                if utterance_options.trailing_silence
+                else 0.35,
+                text="",
+            )
+        if is_given(format):
+            self._opts.format = format
+        if is_given(context):
+            self._opts.context = context
+        if is_given(split_utterances):
+            self._opts.split_utterances = split_utterances
+        if is_given(num_generations):
+            self._opts.num_generations = num_generations
+        if is_given(instant_mode):
+            self._opts.instant_mode = instant_mode
+        if is_given(strip_headers):
+            self._opts.strip_headers = strip_headers
+
+    def synthesize(
+        self,
+        text: str,
+        *,
+        conn_options: APIConnectOptions = DEFAULT_API_CONNECT_OPTIONS,
+    ) -> ChunkedStream:
+        return ChunkedStream(
+            tts=self,
+            input_text=text,
+            conn_options=conn_options,
+            opts=self._opts,
+        )
+
+    def stream(
+        self,
+        *,
+        conn_options: APIConnectOptions = DEFAULT_API_CONNECT_OPTIONS,
+    ) -> SynthesizeStream:
+        stream = SynthesizeStream(
+            tts=self,
+            conn_options=conn_options,
+            opts=self._opts,
+        )
+        self._streams.add(stream)
+        return stream
+
+    async def aclose(self) -> None:
+        """Close all streams and cleanup resources."""
+        for stream in list(self._streams):
+            await stream.aclose()
+        self._streams.clear()
+        await super().aclose()
+
+
+class ChunkedStream(tts.ChunkedStream):
+    """Stream for Hume TTS JSON streaming API."""
+
+    def __init__(
+        self,
+        *,
+        tts: TTS,
+        input_text: str,
+        opts: _TTSOptions,
+        conn_options: APIConnectOptions,
```

Comment:
Can we make sure the timeout inside conn_options is respected, otherwise the tests will not pass